File: src/ts/magma/api/text/Characters.ts
import { Actual } from "../../../magma/annotate/Actual";
import { Namespace } from "../../../magma/annotate/Namespace";
export interface CharactersInstance {
	isDigit(c: string): boolean;

	isLetter(c: string): boolean;

}
export declare const Characters: CharactersInstance;

File: src/java/magma/api/text/Characters.java
package magma.api.text;

import magma.annotate.Actual;
import magma.annotate.Namespace;

@Namespace
public class Characters {
    @Actual
    public static boolean isDigit(char c) {
        return Character.isDigit(c);
    }

    @Actual
    public static boolean isLetter(char c) {
        return Character.isLetter(c);
    }
}
